sdk/go/sdk.go

```diff
@@ -79,10 +79,10 @@ type User struct {
 
 // Thing represents mainflux thing.
 type Thing struct {
-	ID       string                 `json:"id,omitempty"`
-	Name     string                 `json:"name,omitempty"`
-	Key      string                 `json:"key,omitempty"`
-	Metadata map[string]interface{} `json:"metadata,omitempty"`
+	ID       string                 `json:"id,omitempty" csv:"id,omitempty"`
+	Name     string                 `json:"name,omitempty" csv:"name,omitempty"`
+	Key      string                 `json:"key,omitempty" csv:"key,omitempty"`
+	Metadata map[string]interface{} `json:"metadata,omitempty" csv:"-"`
```

Comment:
Why are you ignoring metadata? Doesn't look nice in the CSV file? If so, maybe use JSON instead of CSV.

Reply:
Good catch.  The CSV library doesn't seem to support `map` types ATM.  I planned on looking into writing a marshal/unmarshal function for it.  This reminded me to do so.